src/styleguide/components/s-api-mock-test.vue

```diff
@@ -51,19 +51,22 @@
               {{ endpoint.path }}
             </td>
             <td>
-              <e-checkbox v-model="configurations[endpoint.header].enabled"
+              <!-- eslint-disable-next-line vue/valid-v-model, vue/no-extra-parens -->
+              <e-checkbox v-model="(configurations[endpoint.header] as Record<'enabled', boolean>).enabled"
```

Comment:
🙈 Probably not worth investing time to solve it differently since only styleguide functionality.